crane-cloud/frontend
# Add environment variables for docs and blog

# Description
Fix broken docs link on production and staging

## Type of change


## Trello Ticket ID

Please add a link to the Trello ticket for the task.

## How Can This Been Tested?
Checkout docs on the staging/prod and you will notice they are broken, this PR fixes that

## Checklist:
